x-pack/plugins/cases/public/components/filter_popover/index.test.tsx

```diff
@@ -6,31 +6,21 @@
  */
 
 import React from 'react';
-import { waitForEuiPopoverOpen, screen } from '@elastic/eui/lib/test/rtl';
+import { waitForEuiPopoverOpen, screen, render } from '@elastic/eui/lib/test/rtl';
```

Comment:
should `screen` and `render` be from `'@testing-library/react'` instead? Im not familiar with this eui package

Reply:
You are right! I missed that.